index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet"/>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <title>NIGERIA MUSIC SUMMIT</title>
+</head>
+<body>
+  <header>
+    <aside class="top-section">
+      <div class="bar-container">
+      <ul class="bars">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
+        </li>
+      </ul>
+      </div>
+    </aside>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="./image/logo.jpg" alt="LOGO" class="logo"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Securam Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+
+<section class="home-page">
+  <div class="title">
+    <h1 class="intro-headline">"Hello!! It's Afrobeat to the world"</h1>
+    <h1 class="main-headline" >NIGERIA MUSIC SUMMIT 2021</h1>
+    <div class="theme">
+      <p>
+        Celebration of Nigerian hiphop miusic, its global reception, and challenges. creating a positive change in Nigerian music industry.
+      </p>
```

Comment:
- [ ] Make the border color white here to match the given design

Yours:
![Screenshot from 2021-11-18 19-15-20](https://user-images.githubusercontent.com/62705947/142473366-dbaa42e9-9d00-4cfc-93b3-34da858a4e28.png)
Original
![Screenshot from 2021-11-18 19-15-22](https://user-images.githubusercontent.com/62705947/142473390-3f40c510-9f4a-49cb-b6be-3f51dfa876a9.png)

Reply:
Well done